app/forms/address_form.rb

```diff
@@ -33,7 +33,15 @@ class AddressForm < Form
   def save
     return false unless valid?
 
-    update!(attributes)
+    journey_session.answers.assign_attributes(
+      address_line_1: address_line_1,
+      address_line_2: address_line_2,
+      address_line_3: address_line_3,
+      address_line_4: address_line_4,
+      postcode: postcode
+    )
```

Comment:
Suggestion: This would be consistent with https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2777/files#diff-eac4d675a6e2dfa63e98a42d3ed7f820c561332eeb7cfe3d63219bcbfeea4741R11

```suggestion
    journey_session.answers.assign_attributes(address_line_1:, address_line_2:, address_line_3:, address_line_4:, postcode:)
```